ui/DesignSystem/CopyableIdentifier.svelte

```diff
@@ -10,6 +10,7 @@
     | "radicleId"
     | "deviceId"
     | "seedAddress"
+    | "seedHost"
```

Comment:
We can revert the changes in this file. We're only using the `seedAddress` kind.

Reply:
Yeah you're right, removed `seedHost` from the component